ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -89,7 +89,7 @@ func startServiceCommand(factory composeFactory.ProjectFactory) cli.Command {
 func upServiceCommand(factory composeFactory.ProjectFactory) cli.Command {
 	return cli.Command{
 		Name:   "up",
-		Usage:  "Creates an ECS service from your compose file (if it does not already exist) and runs one instance of that task on your cluster (a combination of create and start). This command updates the desired count of the service to 1.",
+		Usage:  "Creates or Updates an ECS Service using your compose file. If the service is being created or has its desired count set to 0, then the desired count is set to 1. Otherwise, the service’s task definition is updated to reflect any changes to the compose file, and the desired count remains the same.",
```

Comment:
How about this:

Creates a new ECS service or updates an existing one according to your compose file. For new services or existing services with a current desired count of 0, the desired count for the service is set to 1. For existing services with non-zero desired counts, a new task definition is created to reflect any changes to the compose file and the service is updated to use that task definition. In this case, the desired count does not change.